src/main/java/org/janelia/saalfeldlab/n5/googlecloud/GoogleCloudStorageKeyValueAccess.java

```diff
@@ -123,6 +135,12 @@ private void createBucket() {
 		if (!bucketExists) {
 			try {
 				storage.create(BucketInfo.of(bucketName));
+			} catch (StorageException e) {
+				if (e.getCode() == ALREADY_EXISTS)
```

Comment:
Were you able to actually trigger this? It look like it only gets here if `!bucketExists` so I'm wondering if the ALREADY_EXISTS branch here is possible

Reply:
we changed how this works in the commit below